### DISCUSSION TITLE
How can i use this with a SwiftUI package?

### DISCUSSION THREAD

**Discussion by lperezdude** [pionl/laravel-chunk-upload#157] [General]
If there inst any how can I build one myself what standards should i follow ti interface with it? 
Thank you

**Comment by pionl** [comment]
Pick what every HTTP request format you like based on JS implementation.

I think CONTENT range header implementation is the way to go?

https://github.com/pionl/laravel-chunk-upload/blob/master/src/Handler/ContentRangeUploadHandler.php
